lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -183,10 +182,10 @@ func (ic *ICache) Get(idOrRef string) (*metadata.ImageConfig, error) {
 	return copyImageConfig(config), nil
 }
 
-func (ic *ICache) getImageByDigest(digest digest.Digest) *metadata.ImageConfig {
+func (ic *ICache) getImageByDigest(digest string) *metadata.ImageConfig {
```

Comment:
is this change to signature needed - from just this commit it looks like the original code cast it to string where needed and it's got to be a string type anyway given the comparison on line 167